_data/food.yml

```
@@ -34,10 +34,18 @@ websites:
       tfa: No
       twitter: tacobell
       facebook: tacobell
+  
+    - name: Wolt
+      url: https://wolt.com/
+      img: wolt.png
+      tfa: No
+      twitter: woltapp
+      facebook: woltapp
+      email_address: info@wolt.com
```

info@ seems to be a general inquiry address, support@ would probably fit better.  _(source: [wolt.com/help#contact-support](https://wolt.com/help#contact-support))_ 
```suggestion
      email_address: support@wolt.com
```